packages/lib-classifier/src/plugins/tasks/SimpleDropdownTask/models/SimpleDropdownAnnotation.js

```diff
@@ -0,0 +1,18 @@
+import { types } from 'mobx-state-tree'
+import Annotation from '../../models/Annotation'
+
+const SimpleDropdown = types.model('SimpleDropdown', {
+  value: types.optional(types.frozen({
```

Comment:
Yes, I'm using `types.optional(types.frozen({ ... }), null)` here.

I have no idea why `types.maybeNull(types.frozen({ ... }))` doesn't work. If I use maybeNull, then if I submit a blank blank (i.e. I don't select any value from the dropdown), the annotation becomes `{ value: nonsense, option: nonsense } ` instead of `null`

Reply:
`types.frozen` is not what should be used here. It's an immutable object, so you can't have an immutable object and maybe have it be null. In addition, this isn't going to work for the annotation value because we need it to be mutable. My suggestion:
```
value: types.maybeNull(types.model({
  option: types.boolean,
  selection: types.number
}))
```

I'd recommend using `suggestion` because `value.value` isn't great for comprehensibility here.

Reply:
Ah, use `types.model`! Thanks Sarah!

Annotations will now be `{ option: bool, selection: number }`